src/index.js

```diff
@@ -1,36 +1,28 @@
-import 'babel-polyfill'
+import '@babel/polyfill'
 import 'whatwg-fetch'
 import Raven from 'raven-js'
 import RootComponent from 'components/Root'
-import { initProviders } from 'integrations/store/actions'
-import { initReadOnlyGnosis } from 'store/actions/blockchain'
+import { initReadOnlyGnosis, requestTargetNetworkId } from 'store/actions/blockchain'
 import Decimal from 'decimal.js'
 import React from 'react'
 
 import ReactDOM from 'react-dom'
 import 'scss/style.scss'
-import { isFeatureEnabled, getProviderConfig } from 'utils/features'
+import store from 'store'
 
-import { WALLET_PROVIDER } from 'integrations/constants'
-
-import store from './store'
 import { setMomentRelativeTime, setMomentDurationFormat } from './setup'
 
-const providerConfig = getProviderConfig()
-const tournamentEnabled = isFeatureEnabled('tournament')
-
 setMomentRelativeTime()
 setMomentDurationFormat()
 
 // load data from localstorage
 store.dispatch({ type: 'INIT' })
 store.dispatch(initReadOnlyGnosis())
-if (!tournamentEnabled) {
-  store.dispatch(initProviders())
-} else {
-  const tournamentProvider = WALLET_PROVIDER[providerConfig.default]
-  store.dispatch(initProviders({ providers: [WALLET_PROVIDER.REMOTE, tournamentProvider] }))
-}
+store.dispatch({ type: 'CHECK_AVAILABLE_PROVIDERS' })
```

Comment:
I'd like to move stuff like this to `integrations/index.js`, which imports the store too and does it's dispatches. Wdyt?